Misc/NEWS.d/next/Core and Builtins/2020-06-20-17-00-44.bpo-35975.UDHCHp.rst

```diff
@@ -0,0 +1,3 @@
+Stefan Behnel reported that cf_feature_version is used even when
+PyCF_ONLY_AST is not set. This is against the intention and against the
+documented behavior, so I fixed it.
```

Comment:
is it possible to avoid "I" in the NEWS entry? At least, replace it with Guido. Otherwise, it's unclear who is "I".

Reply:
I'll change this to "so it's been fixed."

Reply:
Thanks :-)